crates/uv/src/cli.rs

```diff
@@ -1457,6 +1458,31 @@ pub(crate) struct VenvArgs {
     pub(crate) compat_args: compat::VenvCompatArgs,
 }
 
+#[derive(Parser, Debug, Clone)]
+pub(crate) enum ExternalCommand {
+    #[command(external_subcommand)]
+    Cmd(Vec<OsString>),
+}
+
+impl Deref for ExternalCommand {
+    type Target = Vec<OsString>;
+
+    fn deref(&self) -> &Self::Target {
+        match self {
+            Self::Cmd(cmd) => cmd,
+        }
+    }
+}
+
+impl ExternalCommand {
+    pub(crate) fn split(&self) -> (Option<&OsString>, &[OsString]) {
+        match self.deref().as_slice() {
```

Comment:
It is odd to need to use `.deref()` explicitly. Does just `self.as_slice()` work?

Reply:
Probably just me being dumb, thanks!